### ISSUE TITLE
Error creating mixin on 1.20.1

### ISSUE THREAD

**Issue by alexweber** [Erb3/NoHotbarLooping#1]
Hi, thanks for the mod!  Unfortunatly it's broken on 1.20.1:

Here's the simplified trace:

```
java.lang.RuntimeException: Error creating Mixin config nohotbarlooping.mixins.json for mod nohotbarlooping
...
Caused by: org.spongepowered.asm.launch.MixinInitialisationError: Error initialising mixin config nohotbarlooping.mixins.json
...
Caused by: java.lang.IllegalArgumentException: The specified resource 'nohotbarlooping.mixins.json' was invalid or could not be read
``` 

As far as the actual mixin logic goes it's probably still good... :)

I'm not sure what the deal with the config is, whether it's really case sensitive or whether there's another issue...

**Comment by Erb3** [comment]
This is very strange behaviour, and I have no clue what is happening, but I am blaming IntelliJ / git for now. I am looking into it, but I want to give a huge thanks to you for reporting this! I test before releasing, but git is having some weird desync.

**Comment by Erb3** [comment]
I've released version [v1.1.1](https://modrinth.com/mod/nohotbarlooping/version/1.1.1), which should fix your issue. I have no clue what happened, but it should work now! It was working on my machine, which is why I released the previous build... Anyway, if you still have the issue feel free to re-open this issue. 😀

**Comment by alexweber** [comment]
@Erb3 Awesome, thanks for being receptive and for the quick fix!  I just tried it out and it works again! 😄
